day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,37 @@
+# Create a method for decided what to output for each number
```

Comment:
Nice Fizzbuzz! I love that your accept user input and broke up your program into a method. If you have time this weekend then give a few of the other Day-7 challenges a go! You can time box them, so that you don't spend too much time on them, and you get some practice with timed challenges, which you will see weekly at Turing. Great job on the capstone Ryan!